node/src/components/chain_synchronizer/operations.rs

```diff
@@ -1297,81 +1557,134 @@ async fn fetch_block_worker(
     }
 }
 
-/// Runs the chain synchronization task.
-pub(super) async fn run_chain_sync_task(
-    effect_builder: EffectBuilder<JoinerEvent>,
+/// Runs the initial chain synchronization task ("fast sync").
```

Comment:
What does the `initial` mean here? The sync process started in the joiner?

Reply:
Yes, exactly.